### PR TITLE
Add app.devsuite.Manuals

### PR DESCRIPTION
**Pull request by chergert**
Manuals is the documentation component of `org.gnome.Builder` extracted into a standalone application. It not only reads and searches documentation, but allows installing and indexing Flatpak-based documentation SDK extensions such as `org.gnome.Sdk.Docs`.

This will require an exception just like `org.gnome.Builder` as it can install additional SDKs using `libflatpak`.

### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: app.devsuite.Manuals.json [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/polkit-1",
+        "/share/vala",
+        "/share/zsh"
+    ],
+    "modules" : [
+        "python3-pyparsing.json",
+        {
+            "name" : "intltool",
```
Comment (hfiguiere) [inline app.devsuite.Manuals.json:60]: the shared module should be used.

Comment (chergert) [reply app.devsuite.Manuals.json]: Fixed

**Comment by bbhtt** [comment]
bot, build app.devsuite.Manuals

**Comment by bbhtt** [comment]
Should this be bundling `jhbuild`? If it is not installed on host, it seems to fail silently.

**Comment by bbhtt** [comment]
I can't get jhbuild to work on Fedora 40, it is trying to install libmpdsharing-3.0 which does not exist for Fedora 40. So can't test what that part is doing.

**Comment by chergert** [comment]
There is a small fix for that but it should work fine regardless. We could do what builder does and try to use `which` to see if `jhbuild` is available, but it works just fine to try and bail if it is not there.

**Comment by chergert** [comment]
bot, build app.devsuite.Manuals

**Comment by chergert** [comment]
Updated build seems to be fine

**Comment by chergert** [comment]
Anything left to do here? Seems like it's ready to go?

**Pull request closed without merging**